modules/administration/pages/tshoot-firewalls.adoc

```diff
@@ -0,0 +1,47 @@
+[[troubleshooting-firewalls]]
+= Troubleshooting Firewalls
+
+////
+PUT THIS COMMENT AT THE TOP OF TROUBLESHOOTING SECTIONS
+
+Troubleshooting format:
+
+One sentence each:
+Cause: What created the problem?
+Consequence: What does the user see when this happens?
+Fix: What can the user do to fix this problem?
+Result: What happens after the user has completed the fix?
+
+If more detailed instructions are required, put them in a "Resolving" procedure:
+.Procedure: Resolving Widget Wobbles
+. First step
+. Another step
+. Last step
+
+
+Cause: User firewall is set to block outgoing traffic by dropping the packet request. During sync with SCC, SUMA waits for an answer on each URL until it times out, eventually causing the entire refresh of the product list to timeout. Applies only to third party (non-SUSE) products, as sync with SCC needs to access locations other than SCC to verify if the if the download location is valid.
+
+Consequence: The sync to SCC fails, and the third party products are not shown in the product list.
+
+Fix: Configure the firewall to reject requests from SUMA instead of drop (preferred), or configure a firewall on the server (if no ability to change firewall settings)
+
+Result: Sync to SCC will either be able to reach the URLs required, or will have request rejected so that the request fails rather than times out.
+////
+
+If you are using a firewall that blocks outgoing traffic, it will either ``REJECT`` or ``DROP`` network requests.
+If it is set to ``DROP`` then you might find that synchronizing with the {scc} times out when you are using non-{suse} clients, including {centos}, {ubuntu}, and {oracle}.
+
+This occurs because the synchronization process for non-{suse} clients needs to access the third-party repositories that provide packages for these clients, and not just the {scc}.
```

Comment:
The reason is, that we "test" access to these external repos. The product page should only show products where the customer has access to. That requires to do a http request to "all" repositories  we can support.